manticore/platforms/linux.py

```diff
@@ -1719,6 +1720,8 @@ def sys_mmap(self, address, size, prot, flags, fd, offset):
 
         perms = perms_from_protflags(prot)
 
+        cpu._publish('will_map_memory', address, size, perms, "" if fd == 0 or flags & 0x20 else self.files[fd].name, offset)
```

Comment:
What does  `flags & 0x20` check for? (might be good to add a comment)

Any chance to change `"" if fd == 0 or flags & 0x20 else self.files[fd].name` to just `self.files[fd].name` and make it so we set a name on stdin/out/err manually?

Hmm, stdout/stderr/in shall have names actually:
```
In [4]: sys.stdout
Out[4]: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='UTF-8'>

In [5]: sys.stdout.name
Out[5]: '<stdout>'